spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
If you're having trouble with some validations or associations, in our case, we had to disable the randomness of tests because it affected some of our tests with the `let` statements because they are `lazy loaded`. If you'd like them to be loaded in a given order or obligatory then you need to use `let!`.

[OPTIONAL]